crates/allocator/src/lib.rs

```diff
@@ -23,8 +23,21 @@
 // We use `wee_alloc` as the global allocator since it is optimized for binary file size
 // so that contracts compiled with it as allocator do not grow too much in size.
 #[cfg(not(feature = "std"))]
+#[cfg(feature = "wee-allocator")]
+#[cfg(not(feature = "bump-allocator"))]
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
+#[cfg(not(feature = "std"))]
+#[cfg(feature = "bump-allocator")]
+#[cfg(not(feature = "wee-allocator"))]
+#[global_allocator]
+static mut ALLOC: bump::BumpAllocator = bump::BumpAllocator {};
+
 #[cfg(not(feature = "std"))]
 mod handlers;
+
+#[cfg(not(feature = "std"))]
+#[cfg(feature = "bump-allocator")]
+#[cfg(not(feature = "wee-allocator"))]
```

Comment:
We really should remove the `bump-allocator` feature and make it our default and make `wee-alloc` an opt-in instead of this where we would always require one of the features of be enabled.